### PR TITLE
feat: add ticketing refund forms 

### PR DESCRIPTION
**Pull request by jonasbrunvoll**
This PR provides refund forms, under the ticketing page.

### CODE REVIEW

File: src/page-modules/contact/ticketing/events.ts [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
-      value: string;
+      inputName:
+        | 'question'
+        | 'orderId'
+        | 'customerNumber'
+        | 'refundReason'
+        | 'ticketType'
+        | 'amount'
+        | 'travelCardNumber'
+        | 'isIntialAgreementChecked'
```
Comment (adriansberg) [inline src/page-modules/contact/ticketing/events.ts:24]:
```suggestion
        | 'isInitialAgreementChecked'
```

File: src/translations/pages/contact.ts [hunk 2]
```diff
@@ -1550,9 +1565,54 @@ export const Contact = {
...
+          id: 'periodTicket',
+          name: _(
+            'Periodebillet (sonebasert)',
+            'Period ticket (zone-based)',
+            'Periodebillet (sonebasert)',
+          ),
+        },
+        {
+          id: 'framYoung',
+          name: _('FRAM Ung', 'FRAM Young', 'FRAM Ung'),
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:1592]: At the moment, all of the FRAM tickets have the same name in any language in the app at least, should that be the case here as well?

File: src/translations/pages/contact.ts [hunk 3]
```diff
@@ -1550,9 +1565,54 @@ export const Contact = {
...
+          id: 'framStudent',
+          name: _('FRAM Student', 'FRAM Student', 'FRAM Student'),
+        },
+        {
+          id: 'framAdult',
+          name: _('FRAM Voksen', 'FRAM Adult', 'FRAM Voksen'),
+        },
+        {
+          id: 'framSenior',
+          name: _('Fram Honnør', 'Fram Senior', 'Fram Honnør'),
```
Comment (adriansberg) [inline src/translations/pages/contact.ts:1604]: Capitalize "Fram"

File: src/translations/pages/contact.ts (hunk 2 above)
Comment (jonasbrunvoll) [reply src/translations/pages/contact.ts:1602]: I´ll update the naming👍

File: src/translations/pages/contact.ts (hunk 3 above)
Comment (jonasbrunvoll) [reply src/translations/pages/contact.ts:1614]: 👍

**Review by mortennordseth — APPROVED** [review]

**Pull request merged**